weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/CallsTableButtons.tsx

```diff
@@ -268,6 +330,31 @@ const DownloadGrid: FC<{
           Export to JSON
         </ClickableOutlinedCardWithIcon>
       </div>
+      <div className="mt-8 flex items-center">
+        <ClickableOutlinedCardWithIcon
+          iconName="python-logo"
+          disabled={false}
+          onClick={() => setCodeMode('python')}>
+          Use Python
+        </ClickableOutlinedCardWithIcon>
+        <div className="ml-8" />
+        <ClickableOutlinedCardWithIcon
+          iconName="code-alt"
+          disabled={false}
+          onClick={() => setCodeMode('curl')}>
+          Use CURL
+        </ClickableOutlinedCardWithIcon>
+      </div>
+      {codeMode && (
+        <div className="mt-8 flex max-w-full items-center">
+          <CodeEditor
```

Comment:
I don't know how to properly style this... Might have to break out the component and weave-ify it?

Reply:
How do you want to style it?